common/decisions/wc_culture_decisions.txt

```diff
@@ -0,0 +1,73 @@
+﻿embrace_void_elf_culture_decision = {
+	picture = "gfx/interface/illustrations/decisions/decision_realm.dds"
+	major = yes
+
+	ai_check_interval = 36
+
+	desc = embrace_void_elf_culture_decision_desc
+
+	is_shown = {
+		is_landed = yes
+		can_become_void_elf_trigger = yes
+		NOT = {
+			is_target_in_global_variable_list = {
+				name = unavailable_unique_decisions
+				target = flag:embrace_void_elf_culture_decision
+			}
+		}
+	}
+
+	is_valid = {
+		custom_description = {
+			text = capital_in_quelthalas_region_tooltip
+			capital_province = {
+				geographical_region = world_eastern_kingdoms_quelthalas
+			}
+		}
+		NOT = { culture = culture:void_elf }
+		culture = { has_cultural_pillar = heritage_highborne }
+		has_trait = being_void
+		has_title = title:k_quelthalas
+		OR = { 
+			has_title = title:c_sunwell_plateau
+			any_vassal_or_below = {
+				has_title = title:c_sunwell_plateau
+			}
+		}
```

Comment:
the title requirements are too onerous. in game, the void elves dont even have a title. it's challenging enough to become a void being as it is. i think they should be at least a duke.